packages/cli/src/sso/saml/saml.service.ee.ts

```diff
@@ -138,14 +141,14 @@ export class SamlService {
 
 	private getRedirectLoginRequestUrl(relayState?: string): BindingContext {
 		const sp = this.getServiceProviderInstance();
-		sp.entitySetting.relayState = relayState ?? getInstanceBaseUrl();
+		sp.entitySetting.relayState = relayState ?? this.urlService.editorBaseUrl;
```

Comment:
same here. I'm wondering if we should add `createBackendUrl` and `createFrontendUrl` methods in `URLService`, so that people are forced to declare if the generated url is meant to be handled by the frontend, or the backend.